lib/ts/recipe/emailpassword/components/themes/translations.ts

```diff
@@ -35,9 +35,13 @@ export const defaultTranslationsEmailPassword = {
 
         EMAIL_PASSWORD_RESET_HEADER_TITLE: "Reset your password",
         EMAIL_PASSWORD_RESET_HEADER_SUBTITLE: "We will send you an email to reset your password",
-        EMAIL_PASSWORD_RESET_SEND_SUCCESS: "Please check your email for the password recovery link. ",
-        EMAIL_PASSWORD_RESET_RESEND_LINK: "Resend",
+        EMAIL_PASSWORD_RESET_SEND_SUCCESS_FALLBACK:
```

Comment:
It's fine as is, but this could've been just a fallback for the "email address" part since it repeats both the before and after email parts.